apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -485,7 +485,7 @@ const RemixLine: React.FC<RemixLineProps> = ({
                         return null;
                       if (
                         key === "PROBABILISTIC_RANGE" &&
-                        (Math.round(value[0] * 10) < 0 || Math.round(value[1] * 10) < 0)
+                        (Math.round(value[0] * 100) < 0 || Math.round(value[1] * 100) < 0)
```

Comment:
updated values after multiplying rounded value in `line 488` by `100`: 
`value < / = -.006` no values shown
`0 > value > / = -.005` shows `0.0`